### PR TITLE
WIP: First draft of a page with content guidelines

### PR DESCRIPTION
**Pull request by GBKS**
As we make more content public, we will see many diverse writing styles and perspectives. That is great. There are still a few reasons to have some guidelines, so readers can have a smooth experience and don't feel big disconnects whenever they click through to a different page. Here I tried to gather some thoughts on writing that we can iterate on. Let's discuss if this is something we want, and what we may want to include (and not include).

Here's a [Google Doc](https://docs.google.com/document/d/1F8SBUYtVV9TAWxZrePBij-cDfbFiB1nddTFnxLhb2zY/edit?usp=sharing) I shared on the topic in early August. Just for completeness.

### CODE REVIEW

File: guide/content-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+It is OK to show screenshots of other software as reference points. Make sure to indicate the product and version. Avoid negative examples and focus on showcasing good solutions and explain why they are good.
+
+#### Be humble
+
+The guide is a work in progress and needs to evolve to stay relevant. What we write today may not be applicable tomorrow.
+
+#### Write in the open if you can
+
+It is healthy to get early feedback as it may prevent you from going into a wrong direction for too long. There is a whole community around you to help you create better content that helps more people.
+
```
Comment (johnsBeharry) [inline guide/content-guidelines.md:52]: Perhaps we can give instructions on how to do this - by forking the repo and setting up GitHub Pages.

Comment (GBKS) [reply guide/content-guidelines.md:52]: I would put that on a separate page and link to it. I'm pretty sure we'll need a dedicated how-to for Github with screenshots, a video, etc. And maybe also one for Figma if we provide templates and reference Figma files.

**Comment by moneyball** [comment]
ACK I think this is a great start. I support merging it as the initial guidelines and we can iterate from there. Another idea if you think it merits more feedback is to schedule a live review of it and anyone interested can join.

**Comment by GBKS** [comment]
Could you please review and sign off, so I can merge?

**Comment by johnsBeharry** [comment]
ACK

---

FYI this repo doesn't have the merge checks as /Meta does so you don't need any explicit approvals and can go ahead and merge just based on ACKs.

**Pull request merged**